scripts/github-workflow-e2e-matrix.ts

```diff
@@ -0,0 +1,36 @@
+import { spawnSync } from 'child_process';
+
+try {
+  const all = process.argv.includes('--all');
+  const affectedJson = runCommand(
+    'npx',
+    'nx',
+    'print-affected',
+    '--target=build-storybook',
+    all ? ' --all' : ''
+  );
+  const affected = JSON.parse(affectedJson);
+  const projects = affected.tasks
+    .map((task: any) => task.target.project)
```

Comment:
You can pass `--select=tasks.target.project` to the CLI command to return only the project names.
See: https://github.com/blackbaud/skyux-dev-infra/blob/main/lib/test-affected/handler.ts#L41